packages/fury-cli/lib/fury.js

```diff
@@ -175,14 +184,27 @@ class FuryCLI {
   }
 }
 
+function getPackage(name) {
+  // eslint-disable-next-line import/no-dynamic-require, global-require
+  return require(`${name}/package.json`);
+}
+
+function getVersion() {
+  const packages = ['fury-cli'].concat(adapters);
+
+  return packages
+    .map(getPackage)
+    .map(pkg => `${pkg.name} ${pkg.version}`)
+    .join('\n');
+}
 
 if (require.main === module) {
   let input;
   let output;
 
   commander
     .description(pkg.description)
-    .version(pkg.version)
+    .version(getVersion())
```

Comment:
Ideally, I'd have done something like `.option('--version')` and later did a check for `commander.version` and then computed all the versions. However that isn't possible, the `version` option name is handled in a special way inside Commander and thus if you create a version option you cannot use it later on.